### PR TITLE
Update device-control-removable-storage-access-control.md

### PR DESCRIPTION
**Pull request by tewchen**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ Deploy Removable Storage Access Control on Windows 10 devices that have antimalw
 
 - **4.18.2105 or later**: Add Wildcard support for HardwareId/DeviceId/InstancePathId/FriendlyNameId/SerialNumberId, the combination of specific user on specific machine, removeable SSD (a SanDisk Extreme SSD)/USB Attached SCSI (UAS) support
 
+- **4.18.2107 or later**: Add WPD (Iphone/Android/tablet/etc.) support
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
- **4.18.2107 or later**: Add Windows Portable Device (WPD) support
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - Suggestion only.

**Review by denisebmsft — APPROVED** [review]
Thank you @tewchen @yogkumgit @JohanFreelancer9

**Pull request merged**